Updated bluebird to version 3

iranreyes opened the pull request:
This update includes a breaking change in the way that cancel is handling

cheapsteak commented on test.js:
```diff
@@ -2,74 +2,95 @@ var animate = require('./')
+													st.assert(d.value < 100, 'fromTo cancelling works')
+												});
+
+		setTimeout(function(){
```
indentation's a little odd

cheapsteak commented:
Thanks for the PR Iran! That is quite useful

Sorry to bug you about this but could you please do a lint and match the code style to the existing base?
( Tabs instead of spaces, and no semicolons ; - )

iranreyes commented:
@cheapsteak Already done the tabs for spaces, cleaned the semicolons, removed the spaces inside the curly brackets, let dummy variables name and magic numbers like originally was

cheapsteak commented:
@iranreyes I think you went the wrong direction, everything is spaces now!

cheapsteak commented:
also @mattdesl would the be a patch or a major version bump since it breaks compatibility with people currently using gsap-promise with bluebird v2?

mattdesl commented:
Sounds like a major bump since it could break people's code. :smile:

P.S. If you guys wanted, this could conform to Jam3(ish) style and use [semistandard](https://github.com/Flet/semistandard). You can use `semistandard-format` to auto-format all the code at once, and then add it to the `package.json` so that others get the linter on `npm test`. See [here](https://github.com/mattdesl/scrape-scripts/commit/c9d7f65eb6ce7835f00a664f066df0d0fd18427f#diff-b9cfc7f2cdf78a7f4b91a753d10865a2) for reference.

cheapsteak commented:
I forked the PR and made some code style changes:
https://github.com/mattdesl/gsap-promise/pull/7

Closing this one

iranreyes commented:
thx :+1:
